packages/terra-core-docs/src/terra-dev-site/doc/section-header/example/AccordionSectionHeader.jsx

```diff
@@ -14,9 +14,12 @@ const AccordionSectionHeader = () => {
   };
 
   return (
-    <AccoordianExampleTemplate title="I can accordion, click me" heading="Accordion Section Header" sectionHeaderAttrs={sectionHeaderProps}>
-      <p>Accordion1</p>
-      <p>Accordion2</p>
+    <AccoordianExampleTemplate title="Patient is Allergic to:" heading="Accordion Section Header" sectionHeaderAttrs={sectionHeaderProps}>
```

Comment:
A
```suggestion
    <AccoordianExampleTemplate title="Patient Allergy details :" heading="Accordion Section Header" sectionHeaderAttrs={sectionHeaderProps}>
```

Reply:
Updated in https://github.com/cerner/terra-core/pull/3691/commits/84d91fe8dc4fb0fe911ee4b7a7494bc6fd3e6a62